src/auxinfo/proof.rs

```diff
@@ -46,22 +46,27 @@ impl AuxInfoProof {
         p: &BigNumber,
         q: &BigNumber,
     ) -> Result<Self> {
+        let context_pimod = ();
         let mut pimod_transcript = Transcript::new(b"PaillierBlumModulusProof");
+        pimod_transcript.append_message(b"PiMod ProofContext", context_pimod.as_bytes());
         pimod_transcript.append_message(b"Session Id", &serialize!(&sid)?);
         pimod_transcript.append_message(b"rho", &rho);
         let pimod = PiModProof::prove(
             &PiModInput::new(N),
             &PiModSecret::new(p, q),
+            &context_pimod,
             &mut pimod_transcript,
             rng,
         )?;
-
+        let context_pifac = ();
```

Comment:
@marsella the CI is complaining over here because the let binding has a unit value. How would you like us to handle this until we actually fill out the context ?

Reply:
Can you take it as a parameter here and pass the AuxInfoParticipant's context where you call this function?